app/props/[year]/page.tsx

```diff
@@ -16,14 +16,13 @@ export default async function Page(
   const allowEdits = user?.is_admin || false;
   const years = await getPropYears();
   years.sort((a, b) => b - a);
-  // Add the "next" year - one after the last year in the list.
-  years.unshift(years[0] + 1);
-  const propsAndResolutions = await getProps({ year });
+  // Passing null as the userId gets us only public props.
+  const propsAndResolutions = await getProps({ year, userId: null });
   return (
     <main className="flex flex-col items-center justify-between py-8 px-8 lg:py-12 lg:px-24">
       <div className="w-full max-w-3xl">
         <PageHeading
-          title="Props"
+          title="Public Props"
```

Comment:
Users don't even have direct access to the Props page today (though they could access it if they had the URL), so this doesn't impact them